THIRD-PARTY.md

```diff
@@ -93,21 +90,193 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-
--------------------------------------------------------------------------
+---
 
 Public domain
 
-* The simplex noise implementation is Copyright (c) Stefan Gustavson.
+- The simplex noise implementation is Copyright (c) Stefan Gustavson.
   It bears the following notice:
 
   This library is public domain software, released by the author
   into the public domain in February 2011. You may do anything
   you like with it. You may even remove all attributions,
   but of course I'd appreciate it if you kept my name somewhere.
 
-
--------------------------------------------------------------------------
+---
+
+                               Modified Apache 2.0 License
+
+TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
We need to say here WHICH parts of our code (and which codebases we derived from) use this reproduced license, just like in all the sections above. So something like:

Modified version of Apache-2.0

* Universal Scene Description
  Copyright Pixar.

  Portions of the build_osl.py were derived from scripts distributed with USD.

... then you reproduce the license ...

Reply:
@lgritz I copied this part from the USD file, lots of lines of build_usd.py is edited. don't know how to mention all the edited lines tbh.

Reply:
Do just like I said above -- insert those few lines right above the license which you reproduced.